packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -231,58 +235,14 @@ if (answers.isUsingTypeScript) {
     // specFileRetriesDeferred: false,
     //
-    <%- include('snippets/reporters', { reporters: answers.reporters }) %>
-    <% if(answers.framework === 'mocha') { %>
-    //
-    // Options to be passed to Mocha.
-    // See the full list at http://mochajs.org/
-    mochaOpts: {
-        ui: 'bdd',
-        timeout: 60000
-    },<%
-    }
-    if(answers.framework === 'jasmine') { %>
-    //
-    // Options to be passed to Jasmine.
-    jasmineOpts: {
-        // Jasmine default timeout
-        defaultTimeoutInterval: 60000,
-        //
-        // The Jasmine framework allows interception of each assertion in order to log the state of the application
-        // or website depending on the result. For example, it is pretty handy to take a screenshot every time
-        // an assertion fails.
-        expectationResultHandler: function(passed, assertion) {
-            // do something
-        }
-    },
-    <% }
-    if(answers.framework === 'cucumber') { %>//
-    // If you are using Cucumber you need to specify the location of your step definitions.
-    cucumberOpts: {
-        // <string[]> (file/dir) require files before executing features
-        require: ['<%- answers.stepDefinitions %>'],
-        // <boolean> show full backtrace for errors
-        backtrace: false,
-        // <string[]> ("extension:module") require files with the given EXTENSION after requiring MODULE (repeatable)
-        requireModule: [],
-        // <boolean> invoke formatters without executing steps
-        dryRun: false,
-        // <boolean> abort the run on first failure
-        failFast: false,
-        // <boolean> hide step definition snippets for pending steps
-        snippets: true,
-        // <boolean> hide source uris
-        source: true,
-        // <boolean> fail if there are any undefined or pending steps
-        strict: false,
-        // <string> (expression) only execute the features or scenarios with tags matching the expression
-        tagExpression: '',
-        // <number> timeout for step definitions
-        timeout: 60000,
-        // <boolean> Enable this config to treat undefined definitions as warnings.
-        ignoreUndefinedDefinitions: false
-    },
-    <% } %>
+    <%- include('snippets/reporters', { reporters: answers.reporters })
+    %><%- answers.serenityAdapter   === 'mocha'    ? include('snippets/serenity-js/mocha', { answers }) : ''
+    %><%- answers.framework         === 'mocha'    ? include('snippets/mocha', { answers }) : ''
+    %><%- answers.serenityAdapter   === 'jasmine'  ? include('snippets/serenity-js/jasmine', { answers }) : ''
+    %><%- answers.framework         === 'jasmine'  ? include('snippets/jasmine', { answers }) : ''
+    %><%- answers.serenityAdapter   === 'cucumber' ? include('snippets/serenity-js/cucumber', { answers }) : ''
+    %><%- answers.framework         === 'cucumber' ? include('snippets/cucumber', { answers }) : ''
+    %>
```

Comment:
Why splitting between Serenity and standard framework opts? Are they differ by a lot?

Reply:
They differ quite a bit for Cucumber since S/JS supports all the major versions of Cucumber, which means supporting legacy parameters, tag filters from before tag expressions, and so on.

For Mocha and Jasmine, most configuration options are the same, but there are some slight differences, like support for `strict` flag in S/JS Mocha, but not in WDIO, or support for `cleanStack` in WDIO but not in S/JS
- https://serenity-js.org/api/jasmine-adapter/interface/JasmineConfig/ vs https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-jasmine-framework/src/types.ts
- https://serenity-js.org/api/mocha-adapter/interface/MochaConfig/ vs https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-mocha-framework/src/types.ts

Thinking about it, we could try and standardise the adapters in the future 🤔

Reply:
Sure, I can do that.